packages/driver/cypress/e2e/commands/files.cy.js

```diff
@@ -325,11 +325,16 @@ describe('src/cy/commands/files', () => {
         Cypress.backend.resolves(okResponse)
 
         cy.on('fail', (err) => {
-          const { lastLog } = this
+          const { fileLog, logs } = this
 
-          assertLogLength(this.logs, 1)
-          expect(lastLog.get('error')).to.eq(err)
-          expect(lastLog.get('state')).to.eq('failed')
+          const assertLog = logs.find((log) => log.get('name') === 'assert')
+
+          expect(fileLog.get('state')).to.eq('passed')
+          expect(fileLog.get('error')).to.be.undefined
+
+          expect(assertLog.get('name')).to.eq('assert')
+          expect(assertLog.get('error')).to.eq(err)
+          expect(assertLog.get('state')).to.eq('failed')
```

Comment:
`cy.readFile()` resolved correct, the assertion is what failed.

| Before | After |
| - | - |
| <img width="587" alt="Screen Shot 2022-11-03 at 8 23 04 AM" src="https://user-images.githubusercontent.com/14099737/199732285-4f1a0c37-4219-4eb3-9456-2b5e1c8788ac.png">|<img width="584" alt="Screen Shot 2022-11-03 at 8 23 12 AM" src="https://user-images.githubusercontent.com/14099737/199732287-92df8a42-bd13-4495-8df8-ce7a5c36c876.png">